crates/bevy_log/src/lib.rs

```diff
@@ -46,12 +46,14 @@ use tracing_subscriber::{prelude::*, registry::Registry, EnvFilter};
 /// # use bevy_internal::DefaultPlugins;
 /// # use bevy_app::App;
 /// # use bevy_log::LogSettings;
-/// # use bevy_utils::tracing::Level;
+/// # use bevy_utils::tracing::{ Level };
```

Comment:
this looks like something that was part of an experiment and you didn't revert back completely?

Reply:
Yes, sorry about the delay, honestly forgot this PR existed after a while.  That was probably my global config clashing and me not noticing, sorry